wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Message Details/MessageDetailsContentViewController.swift

```diff
@@ -20,6 +20,23 @@ import UIKit
 import WireDataModel
 import WireCommonComponents
 
+// MARK: - MessageDetailsSectionDescription
+
+struct MessageDetailsSectionDescription {
+
+    let headerText: String?
+    let items: [MessageDetailsCellDescription]
+
+    init(
+        headerText: String? = nil,
+        items: [MessageDetailsCellDescription]
+    ) {
+        self.headerText = headerText
+        self.items = items
+    }
```

Comment:
```suggestion
    var headerText: String? = nil
    var items: [MessageDetailsCellDescription]
```

Reply:
We use that initializer in a couple of places..

Reply:
isn't it the same, with the var you get the initializer for free